lib/playload/parser/helpers/create_descriptions.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Parser
+  module Helpers
+    class CreateDescriptions
+      include Import[description_repo: 'repositories.description']
+
+      def call(game_details:, publication:)
+        game_details['game_descriptions'].each do |description_details|
+          existing_description = description_repo.find_by(content: description_details['game_description'])
+          next if existing_description
+
+          create_description(description_details, publication)
```

Comment:
Jut regarding this pattern, will this not attempt to create a new record if the old record is not found? That would be fine if it's a first time import, or if the record matches exactly with an existing one, but if the record has changed slightly then it would create a new one rather than upserting. Do Gameopedia provide any kind of uuid for their objects that we could use to match with imported records in our db (rather than using mutable fields such as a description)?